scenarios/openshift/regex_openshift_pod_kill.yml

```diff
@@ -1,20 +1,5 @@
-config:
-  runStrategy:
-    runs: 1
-    maxSecondsBetweenRuns: 30
-    minSecondsBetweenRuns: 1
-scenarios:
-  - name: kill up to 3 pods in any openshift namespace
-    steps:
-    - podAction:
-        matches:
-          - namespace: "openshift-.*"
-        filters:
-          - property:
-             name: "state"
-             value: "Running"
-          - randomSample:
-              size: 3
-        actions:
-          - kill:
-              probability: .7
+# yaml-language-server: $schema=../plugin.schema.json
+- id: kill-pods
+  config:
+    namespace_pattern: ^openshift-.*$
+    kill: 3
```

Comment:
Can we add the label_selector variable here. Running this out of the box is not working